Fix toggle-navbar button for mobile

christianfosli opened the pull request:
The toggle-navbar button on frontend didn't cause the dropdown to appear.

This makes the button operational!

Note: Had to make an unrelated commit 3ef4f6f to make webpack compile 😕

meling commented:
@christianfosli Thanks for the PR. I've asked Vera to review it, but note that your changes went to the master branch, and our current development had been going on for a long time on another branch. I've now rebased our other branch onto master, so you should now be able to see if your changes are still relevant. Maybe we had already fixed some of the issues. Didn't check yet.

christianfosli commented:
Thanks @meling 
I have verified the issue still exists on latest master, and rebased this branch on that.
I dropped commit 3ef4f6f as that was no longer needed.
